### DISCUSSION TITLE
安装的时候样式加载不出来

### DISCUSSION THREAD

**Discussion by JesseBaomihua** [hhyo/Archery#2006] [General]
![image](https://user-images.githubusercontent.com/31915941/208917962-ca4b2545-027e-4ffb-96ff-bcaa11a29f26.png)

![image](https://user-images.githubusercontent.com/31915941/208918098-2ba79ebf-7735-4277-8278-c5e3d83a5367.png)

点击按钮也没有反应，换了mac、ubuntu、win10试了很多遍都是这样

**Comment by igoodful** [comment]
我也是这样的，到底是啥问题呢

**Comment by hhyo** [comment]
#2121
